common/setup_sku_customizations.sh

```diff
@@ -98,20 +95,9 @@ vmSize=\$(echo "\$vmSize" | awk '{print tolower(\$0)}')
 
 ## Topo file setup based on SKU
 case \$vmSize in
-    standard_nc96ads_a100_v4)
-        /opt/azurehpc/customizations/ncv4.sh;;
     
     standard_nd*v4)
```

Comment:
Is this line still useful?

Reply:
let me double check this

Reply:
Same reason you gave above for this removal. It is AMD GPU section.